torch/_dynamo/eval_frame.py

```diff
@@ -790,41 +963,11 @@ def result_capturing_wrapper(*graph_inputs):
     flat_both = list(graph_captured_result) + flat_args
     matched_output_elements_positions = produce_matching(flat_both, flat_results_traced)
 
-    class ChangeInputOutputSignature(torch.fx.interpreter.Transformer):
```

Comment:
No change here right

Reply:
Nope i just moved it out of the export call